sops.go

```diff
@@ -664,6 +665,12 @@ func (m Metadata) GetDataKeyWithKeyServices(svcs []keyservice.KeyServiceClient)
 // returning as soon as one key service succeeds.
 func decryptKeyGroup(group KeyGroup, svcs []keyservice.KeyServiceClient) ([]byte, error) {
 	var keyErrs []error
+
+	// Sort MasterKeys in the group so we try offline ones first
+	sort.SliceStable(group, func(i, j int) bool {
```

Comment:
I'm a bit worried that this modifies the passed-in data. How much work would it be to sort a list of indices instead?

Reply:
@felixfontein you mean to sort a list of indices and then use this list to iterate over group in for loop below? Should be a small change.

Reply:
Yes, that's what I mean.

Reply:
@felixfontein Ok, I've changed it to use indices. Please let know if you have suggestion for better variable names.